From 9261ec504fa6fa3f1b6cbf98bc9b888cf5cf2c71 Mon Sep 17 00:00:00 2001
From: Yan Markman <ymarkman@marvell.com>
Date: Tue, 4 Sep 2018 17:42:23 +0300
Subject: [PATCH 0567/1051] net: mvpp2: increase max number of RX descriptors

This patch is one from set fixing the "rcu_preempt self-detected stall"

Many rx-bm drops and rx-queue-full and tcp retransmissions may be seen
on bi-directional iperf3-TCP started between boards.

Increase max number of RX descriptors

Change-Id: I46f0ae08996d15d6a43a62c20c1fdf4602d06b62
Signed-off-by: Yan Markman <ymarkman@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/59727
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
Tested-by: Stefan Chulski <stefanc@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/net/ethernet/marvell/mvpp2/mvpp2.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/marvell/mvpp2/mvpp2.h b/drivers/net/ethernet/marvell/mvpp2/mvpp2.h
index 9eb383b40818..c5006ed5bf69 100644
--- a/drivers/net/ethernet/marvell/mvpp2/mvpp2.h
+++ b/drivers/net/ethernet/marvell/mvpp2/mvpp2.h
@@ -569,7 +569,7 @@
 #define MVPP2_DEFAULT_RXQ		1
 
 /* Max number of Rx descriptors */
-#define MVPP2_MAX_RXD_MAX		1024
+#define MVPP2_MAX_RXD_MAX		2048
 #define MVPP2_MAX_RXD_DFLT		MVPP2_MAX_RXD_MAX
 
 /* Max number of Tx descriptors */
-- 
2.17.1

